packages/instant/src/types.ts

```diff
@@ -22,12 +22,31 @@ export interface ERC20AssetMetaData {
 export interface ERC721AssetMetaData {
     assetProxyId: AssetProxyId.ERC721;
     name: string;
+    representationUrl?: string;
     primaryColor?: string;
 }
 
 export type AssetMetaData = ERC20AssetMetaData | ERC721AssetMetaData;
 
+export interface ERC20Asset {
+    assetData: string;
+    metaData: ERC20AssetMetaData;
+}
+
+export interface ERC721Asset {
+    assetData: string;
+    metaData: ERC721AssetMetaData;
+}
+export interface Asset {
```

Comment:
minor: would `Asset` being `ERC20Asset | ERC721Asset` be equivalent/easier to read?

Reply:
For reasons that are not entirely clear to me, they are not equivalent and `type Asset = ERC20Asset | ERC721Asset` does not compile when you are trying to create an `Asset` that has a `metaData` field of type `AssetMetaData`

Reply:
Cool w/ leaving as is, not a big deal

Reply:
@BMillman19 we need `ERC20Asset` for the amount input yea